ui/components/Header.js

```diff
@@ -378,7 +378,7 @@ function K8sContextMenu({
                                 />
                               </BadgeAvatars> :
                               <Avatar src="/static/img/kubernetes.svg" className={classes.icon}
-                                style={operStatus ? { margin : 8 } : { opacity : 0.2, margin : 8 }}
```

Comment:
cc: @Abhishek-kumar09 since you added this margin. Wanted to make sure I didn't break the UI in any other case when this margin is needed.